src/Elasticsearch/EndpointAndIndexesConfigurator.php

```diff
@@ -53,55 +41,20 @@ public function createOrUpdate(ConfigReader $configReader): void
             $this->handleAssetIndices($configReader);
         }
 
-        if ($configReader->isObjectIndexingEnabled()) {
+        if ($configReader->isAssetIndexingEnabled()) {
```

Comment:
Why this was changed?

Reply:
I don't see option on frontend for ObjectIndexingEnabled. Already ignored in other places.